ql/src/java/org/apache/hadoop/hive/ql/metadata/Hive.java

```diff
@@ -4123,9 +4125,28 @@ private static void copyFiles(final HiveConf conf, final FileSystem destFs,
           }
           throw new HiveException(e);
         }
-      } else {
+      else {
```

Comment:
nit: I see this kind of formatting very rarely in Hive code

Reply:
Typo, done.